cli/src/commands/list.ts

```diff
@@ -24,35 +23,55 @@ export default class List extends Command {
     }),
   };
 
+  protected getListTable(isLocked: boolean) {
+    const listTable = {
+      columns: ['name', 'updatedAt', 'URL'],
+      config: {
+        name: {
+          headingTransform: () => 'APPLICATION',
+          minWidth: 25,
+        },
+        updatedAt: {
+          headingTransform: () => 'LAST UPDATED',
+          minWidth: 25,
+        },
+      },
+    };
+    if ( isLocked ) {
+      listTable.columns.push('lockReason');
+      const lockTmp = {
+        lockReason: {
+        headingTransform: () => 'LOCK REASON',
+        minWidth: 25,
+      },
+    };
+      return Object.assign(listTable, lockTmp);
```

Comment:
```suggestion
      return { ...listTable, ...lockTmp };
```
We usually avoid `Object.assign`.  That function changes one of the input objects, which can be confusing.